rust_src/src/keymap.rs

```diff
@@ -0,0 +1,41 @@
+use remacs_macros::lisp_fn;
```

Comment:
You are missing the module level comment. First line should be something like `//! Keymap support`.